master/internal/job/job.go

```diff
@@ -102,8 +103,22 @@ func (j *Manager) GetJobs(
 		return nil, err
 	}
 
+	// Try to fetch External jobs, if supported by the Resource Manager (RM).
+	// If the GetExternalJobs call is supported, RM returns a list of external jobs or
+	// an error if there is any problem. Otherwise, RM returns rmerrors.ErrNotSupported
+	// error. In this case, continue without the External jobs.
+	externalJobs, err := j.rm.GetExternalJobs(j.system, sproto.GetExternalJobs{
+		ResourcePool: resourcePool,
+	})
+	if err != nil {
+		// If the error is not 'ErrNotSupported' error, propagate the error upwards.
+		if err != rmerrors.ErrNotSupported {
+			return nil, err
+		}
+	}
```

Comment:
few things in this suggestion
* always compare errors with `errors.Is` so we can wrap errors with context without breakage
* switch is preferred style for checking multiple things from a return
* the comment doesn't add any more information than the code, just drop it
```suggestion
	switch {
	case errors.Is(err, rmerrors.ErrNotSupported):
		break
	case err != nil:
		return nil, err
	}
```